src/components/portal/portal.js

```diff
@@ -12,12 +12,34 @@ class Portal extends React.Component {
      * @property children
      * @type {Node}
      */
-    children: PropTypes.node
+    children: PropTypes.node,
+    onReposition: PropTypes.func
   }
 
   componentWillUnmount() {
     Browser.getDocument().body.removeChild(this.defaultNode);
     this.defaultNode = null;
+    this.scrollParent && this.scrollParent.removeEventListener('scroll', this.props.onReposition);
+    Browser.getWindow().addEventListener('resize', this.props.onReposition);
```

Comment:
`addEventListener`? Think this should be `remove` ;)

Could wrap these in a `if (this.props.onReposition)` check too?